src/main/java/com/twilio/taskrouter/application/servlet/EnqueueServlet.java

```diff
@@ -36,20 +36,23 @@ public EnqueueServlet(TwilioAppSettings twilioSettings) {
   public void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException,
     IOException {
     String selectedProduct = getSelectedProduct(req);
+
     final TwiMLResponse twimlResponse = new TwiMLResponse();
     final Enqueue enqueue = new Enqueue();
     enqueue.setWorkflowSid(workflowSid);
+
     try {
       enqueue.append(new Task(String.format("{\"selected_product\": \"%s\"}", selectedProduct)));
       twimlResponse.append(enqueue);
     } catch (final TwiMLException e) {
       LOG.log(Level.SEVERE, "Error while appending enqueue task to the response", e);
     }
+
     resp.setContentType("application/xml");
     resp.getWriter().print(twimlResponse.toXML());
   }
 
-  public String getSelectedProduct(HttpServletRequest request) {
+  String getSelectedProduct(HttpServletRequest request) {
```

Comment:
Would you mind changing the attribute accessor to _private_. When coding is better to be explicit.

Since this method will be consumed only for this class, we want to make it private. Pretty sure you don't want to make this method _package-private_, right? :wink: